### ISSUE TITLE
Styling of SDSS icon is misleading

### ISSUE THREAD

**Issue by alexbfree** [zooniverse/Galaxy-Zoo#202]
There are three equally sized and styled icons on the classify screen: SDSS, Favourite, and Invert. The right hand two are buttons that respond to a click, and the SDSS button does not. Presumably this is just providing information on the source of the image... as such it should be styled differently from the buttons that are actually clickable - OR, it should be clickable and send you somewhere when you click it (e.g. http://www.sdss.org/ )

**Comment by willettk** [comment]
Yes - it changes based on the telescope used to classify the active data
set (SDSS, UKIRT, Hubble, etc). No objections to a moderate change; just
put a border or something around it? Should ask @heathv what might be best
...

**Comment by heathv** [comment]
Looks like that icon is part of a sprite, one that has a green version of that icon. Easiest solution would be to use that one, but it may still look clickable. Otherwise I'd probably have to make a new icon.

**Comment by vrooje** [comment]
Could it be clickable (or mouseover with no delay, i.e. not just a Title
tag) with a pop-up that explains the SDSS in a sentence?

**Comment by heathv** [comment]
Yeah, a little tooltip could work.

**Comment by willettk** [comment]
I'd vote for clickable rather than hovertext; I think users would get
mildly annoyed if it kept popping up as they moved around the screen.

**Comment by murraycu** [comment]
If it's not clearly a link then the user might confuse it with an action button such as the Invert button. Then they would click it the first time, get taken to a separate web site, and lose their classification in progress. Not the end of the world, but not ideal.

**Comment by heathv** [comment]
Another simple solution would be to remove the icon and add a line of text below the subject explain (very briefly) and linking to the SDSS.

![image](https://cloud.githubusercontent.com/assets/4551411/8307192/ec162032-1982-11e5-9df9-5ea9c8c57ea1.png)

**Comment by willettk** [comment]
I'd be OK with either clickable text popping up (brief explanation +
hyperlink to SDSS site), or the text example above. @heathv and @alexbfree
- what do you think is best from a design/UI perspective?
